doc/user_guide/Running_in_Webassembly.md

```diff
@@ -0,0 +1,176 @@
+
+pn.Row(slider, pn.bind(callback, slider)).servable(target='simple_app');
+```
+
+This code will render this simple application into the `simple_app` DOM node:
+
+```python
+<div id="simple_app"></div>
+```
+
+Alternatively you can also use the `panel.io.pyodide.write` function to write into a particular DOM node:
+
+```python
+await pn.io.pyodide.write('simple_app', component)
+```
+
+### PyScript
+
+Current versions of PyScript will automatically render the output of the last cell of a <py-script> tag. E.g. in this example the `pn.Row()` component will be rendered wherever you placed the tag:
+
+```html
+<py-script>
+  import panel as pn
+
+  slider = pn.widgets.FloatSlider(start=0, end=10, name='Amplitude')
+
+  def callback(new):
+      return f'Amplitude is: {new}'
+
+  pn.Row(slider, pn.bind(callback, slider))
+</py-script>
+```
+
+## Converting Panel applications
+
+Writing an HTML file from scratch with all the Javascript and Python dependencies and other boilerplate can be quite annoying, and requires learning a good bit of HTML. To avoid writing all the boilerplate, Panel provides support for converting an entire application (including Panel templates) to an HTML file, using the `panel convert` API. As a starting point create one or more Python scripts or notebook files containing your application. The only requirement is that they import only global modules (relative imports of other scripts or modules is not supported).
+
+The ``panel convert`` command has the following options:
+
+    positional arguments:
+      SCRIPTs               The scripts or notebooks to convert
+
+    optional arguments:
+      -h, --help            Show this help message and exit
+      --to                  The format to convert to, one of 'pyodide', 'pyodide-worker' or 'pyscript'
+      --out                 Directory to export files to
+      --title               Custom title for the application(s)
+      --prerender           Whether to export pre-rendered models to display while pyodide loads.
+      --index               Whether to create an index if multiple files are served.
+      --pwa                 Whether to add files to allow serving the application as a Progressive Web App.
+      --requirements        Explicit list of Python requirements to add to the converted file.
+
+### Formats
+
+Using the `--to` argument on the CLI you can control the format of the file that is generated by `panel convert`. You have three options, each with distinct advantages and disadvantages:
+
+- **`pyodide`** (default): Run application using Pyodide running in the main thread. This option is less performant than pyodide-worker but produces completely standalone HTML files that do not have to be hosted on a static file server.
+- **`pyodide-worker`**: Generates an HTML file and a JS file containing a Web Worker that runs in a separate thread. This is the most performant option, but files have to be hosted on a static file server.
```

Comment:
Extend line to "static file server like github pages". Users might not know what a static file server means.